sormas-api/src/main/java/de/symeda/sormas/api/Disease.java

```diff
@@ -69,8 +69,9 @@ public enum Disease
 	YAWS_ENDEMIC_SYPHILIS(true, false, false, true, false, 0, true, false, false),
 	MATERNAL_DEATHS(true, false, false, true, false, 0, true, false, false),
 	PERINATAL_DEATHS(true, false, false, true, false, 0, true, false, false),
-	INFLUENZA_A(true, false, true, false, false, 0, true, false, false),
-	INFLUENZA_B(true, false, true, false, false, 0, true, false, false),
+	INFLUENZA(true, true, true, true, false, 0, true, false, false),
```

Comment:
the previous "Influenza_A" and "Influenza_B" were not enabled for aggregate report. The new "Influenza" should be the same.